scripts/Dockerfile

```diff
@@ -65,6 +65,11 @@ RUN set -eux; apt-get update \
   && mv ./rel/couchdb /usr/local \
   && cd / \
   && rm -rf /usr/src/couchdb \
+  && curl -fsSL --compressed https://nodejs.org/dist/v12.9.1/node-v12.9.1-linux-x64.tar.xz \
```

Comment:
The debian package for nodejs is too old?

Reply:
It is just the same version as we have in production